### DISCUSSION TITLE
[v4] export all the functions and types of @tailwindcss/cli

### DISCUSSION THREAD

**Discussion by devcui** [tailwindlabs/tailwindcss#15234] [Ideas]
- Can you export all the functions and types of @tailwindcss/cli?
  - Because I need to make a batch compilation tool, but currently I can only copy the code into my own project
  - But the best way is to directly introduce something like `import {handle,outputFile,the others...} from @tailwindcss/cli`
  
- My project like this
  - https://github.com/rymcu/tailwind-plugin-theme/tree/master
  - input json and build some themes

**Comment by devcui** [comment]
Or the official method provides a way to dynamically switch multiple themes without modifying the code.

**Comment by james598goff** [comment]
Hello,
To export functions [ringsize chart](https://www.ringsize-chart.com) and types from @tailwindcss/cli, you can use the tailwindcss-export-config package. This tool allows you to export Tailwind configurations and use them in your project. 

Best Regards,
James Goff

**Reply by devcui** [reply]
> tailwindcss-export-config

- thanks for your reply
- currently I am using version 4.0.0-beta.3, and tailwind.config.js has been removed.
